src/model/wallet.ts

```diff
@@ -90,7 +91,7 @@ interface WalletInitialized {
 
 interface TransactionRequestParam {
   transaction: TransactionRequest;
-  existingWallet?: Wallet;
+  existingWallet?: Signer;
```

Comment:
our `LedgerSigner` class is the reason for this change, `Wallet` is a implementation of `Signer`